client/pages/home.jsx

```diff
@@ -1,10 +1,97 @@
-import React from 'react';
-import HelloWorld from '../components/hello-world';
+import React, { useState } from 'react';
+// import NavBar from '../components/nav-bar';
+import Banner from '../components/banner';
+import { Carousel } from '../components/carousel';
+import kalopsiaLogo from '../../images/kalopsia-logo.png';
+import kalopsia from '../../images/kalopsia.png';
 
 export default function Home(props) {
+  const [click, setClick] = useState(false);
+  const [searchTerm, setSearchTerm] = useState('');
+
+  function handleClick() {
+    setClick(!click);
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    fetch('/api/sneakers')
+      .then(res => res.json())
+      .then(data => {
+        // eslint-disable-next-line no-console
+        console.log(data.rows);
+      });
+    // eslint-disable-next-line no-console
+    console.log(searchTerm);
+  }
+
+  let showModal;
+  if (click) {
+    showModal = '';
+  } else {
+    showModal = 'hidden';
+  }
+
   return (
     <div>
-      <HelloWorld />
+      {/* <NavBar /> */}
```

Comment:
When you're ready to clean up your code, you should break this into multiple smaller components.